conversion/conversion.go

```diff
@@ -506,19 +506,33 @@ func CreateDatabase(ctx context.Context, adminClient *database.DatabaseAdminClie
 	// Spanner DDL doesn't accept them), and protects table and col names
 	// using backticks (to avoid any issues with Spanner reserved words).
 	// Foreign Keys are set to false since we create them post data migration.
-	schema := conv.SpSchema.GetDDL(ddl.Config{Comments: false, ProtectIds: true, Tables: true, ForeignKeys: false})
-	op, err := adminClient.CreateDatabase(ctx, &adminpb.CreateDatabaseRequest{
+	req := &adminpb.CreateDatabaseRequest{
 		Parent:          fmt.Sprintf("projects/%s/instances/%s", project, instance),
-		CreateStatement: "CREATE DATABASE `" + dbName + "`",
-		ExtraStatements: schema,
-	})
+	}
+	if conv.TargetDb == TARGET_EXPERIMENTAL_POSTGRES {
+		// PG doesn't allow backticks around the database name.
```

Comment:
Suggest rephrasing comment as:
// TARGET_EXPERIMENTAL_POSTGRES doesn't support:
// a) backticks around the database name, or
// b) DDL statements as part of a CreateDatabase operation (so schema
//      must be set using a separate UpdateDatabase operation).

And then deleting the PG comment in the else branch (lin 518).
I think splitting the comments is kind of confusing because both comments are really for this branch.

Reply:
Done.